Header header

bool system_good
bool enable
float64 time_detect
bool[] fields               # [nothing, detect, warning, dangerous]